docs/glossary.md

```diff
@@ -212,39 +224,39 @@ A backwards compatible change to client code which causes upgraded clients to st
 
 ## Staking
 
-"Reserving" tokens (for Polkadot, DOTs) which are put up as "collateral" for a chance to produce a valid block (and thus obtain a block reward). Validators and nominators (who back validators through NPoS) together stake their DOTs in order to add blocks to the relay chain.
+The act of bonding tokens (for Polkadot, DOTs) which are put up as "collateral" for a chance to produce a valid block (and thus obtain a block reward). Validators and nominators stake their DOTs in order to secure the network.
 
 ## State transition function
 
-A function which describes how the state of a blockchain can be transformed. For example, it may describe how tokens can be transferred from one account to another.
+A function that describes how the state of a blockchain can be transformed. For example, it may describe how tokens can be transferred from one account to another.
 
 ## Substrate
 
-An implementation of the Polkadot Runtime Environment which allows developers to generate parachains which are compatible with the Polkadot relay chain.
+A modular framework for building blockchains. Polkadot is build using Substrate. Chains built with Substrate will be easy to connect as parachains.
 
 ## Tabling
 
 In Polkadot governance, bringing a proposal to a vote via referendum. Note that this is the British meaning of "tabling", which is different than the US version, which means "to postpone" a measure.
 
 ## Transaction
 
-An individual element of the state transition function of a block, such as moving tokens from one account to another.
+An extrinsic that is signed. Transactions are gossiped on the network and incur a transaction fee. Transactions are "provably true", unlike inherents. For example, one can prove that Alice wants to send funds to Bob grace of her signature on a transfer-funds message with her private key.
```

Comment:
Bob grace. Something is missing

Reply:
This is a valid sentence, but I understand it might not be natural to everyone.
```suggestion
An extrinsic that is signed. Transactions are gossiped on the network and incur a transaction fee. Transactions are "provably true", unlike inherents. For example, one can prove that Alice wants to send funds to Bob by the fact that she signed a transfer-funds message with her private key.
```